workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/cli.py

```diff
@@ -11,7 +11,9 @@ def dissoc(namespace, key):
 
 def get_parser():
     parser = argparse.ArgumentParser(description="Prognostic run diagnostics")
-    subparsers = parser.add_subparsers(help="Prognostic run diagnostics")
+    subparsers = parser.add_subparsers(
+        help="Prognostic run diagnostics", required=True, dest="command"
```

Comment:
without this, calling `prognostic_run_diags` without any command raises an error instead of printing usage.